ui/packages/components/src/AppRoot/globals.css

```diff
@@ -331,3 +331,40 @@
 #maze-media-host {
   pointer-events: auto;
 }
+
+/*
+ * Legal Disclaimer for Web licence
+ *
+ * Licence to use self-hosted webfonts for displaying dynamic text on specified website domains. Our package includes WOFF and WOFF2 font formats.
+ *
+ * Should the license page views limit be exceeded the license holder will be required to contact Lineto.com for an upgrade.
+ *
+ * It is strictly prohibited to rename the font and to download or use these fonts in any other media.
+ *
+ * These Web fonts are licensed exclusively for the use on the following domain(s) and their subdomains:
+ *
+ * www.inngest.com (monthly pageviews: < 1M)
+ *
+ */
+@font-face {
+  font-family: 'CircularXX';
+  font-style: normal;
+  font-display: fallback;
+  font-weight: 100 400;
+  src: url('https://fonts-cdn.inngest.com/Circular/CircularXXWeb-Regular.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'CircularXX';
+  font-style: normal;
+  font-display: fallback;
+  font-weight: 500 900;
```

Comment:
**question**: Do we only use Medium everywhere? No bold in the app?

What about italic? Should we include that in case we use italic somewhere in copy?